README.md

```diff
@@ -1,123 +1,65 @@
-## Shared storage with S3 backend
-The storage is definitely the most complex and important part of an application setup, once this part is 
-completed, 80% of the tasks are completed.
+以下步驟會透過 s3-fuse 建立 docker image，並讓 k8s pod 能夠 mount AWS s3 bucket。
```

Comment:
This has a bunch of unrelated changes.  Can you narrow this to only the essentials?

Reply:
Oops, my bad.

Reply:
I reverted the un-needed changes.